src/encoded/static/components/search.js

```diff
@@ -786,11 +787,18 @@ class Facet extends React.Component {
         // We have to check the full list for now (until schema change) because some lists contain both numerical and string terms ('Encyclopedia version' under Annotations) and we do not want to sort those by value
         const numericalTest = a => !isNaN(a.key);
         // For date facets, sort by date
-        if (field.match('date') || field.match('month') || field.match('year')) {
-            terms.sort((a, b) => (new Date(b.key) - new Date(a.key)));
+        let terms = [];
+        if (field.match('date')) {
+            terms = _.sortBy(unsortedTerms, obj => moment(obj.key, 'YYYY-MM-DD').utc());
+        } else if (field.match('month')) {
+            terms = _.sortBy(unsortedTerms, obj => moment(obj.key, 'MMMM, YYYY').utc());
```

Comment:
Looks like this format of `moment` returns an object instead of a single value. Maybe add `.toISOString()` to the end to return an ISO-formatted string which is sortable. I _think_ that would work for `sortBy`.

Reply:
This might be clearer than the momentjs docs. https://stackoverflow.com/questions/44813743/returning-a-value-using-moment-js#answer-44813773

Reply:
Somehow, it is sorting the objects, but sorting single values is much clearer.